From 64f10dbc45291f413f128af0428e7209f9b24081 Mon Sep 17 00:00:00 2001
From: sai surya teja <sssteja@habana.ai>
Date: Tue, 07 Mar 2023 07:07:22 +0200
Subject: [PATCH] [SW-72653] gaudi3: ppboot: use bootrom export elf from ppboot repo

This patch modifies gaudi3 ppboot makefile to use bootrom export elf
from ppboot repo instead of arcboot build folder. This is needed as
gaudi3 bootrom development is frozen and the final export elf will be
used in ppboot compilation.

Change-Id: I03e60b4c6982d4efa45c936c15e7b71a439bbd91
---

diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index ae029ac..cdc94c6 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -4,9 +4,11 @@
 PPBOOT_MAX_SIZE := 16384
 PREBOOT_BIN_MAX_SIZE := 130048
 
-# TODO: switch to exports elf provided in preboot sources after gaudi3 bootrom code freeze:
-#	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
-LDFLAGS += -Wl,--just-symbols $(PREBOOT_ROOT_DIR)/../arcboot-custom/exports.elf
+ifeq ($(HL_TARGET),onix)
+	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports_onix.elf
+else
+	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
+endif
 
 CFLAGS += -DPHY_FW_SIZE=$(PHY_FW_SIZE)
 
